From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 10 May 2010 18:50:43 -0400
Subject: [net] e1000/e1000e: compat header cleanup
Message-id: <20100510184828.2781.68507.sendpatchset@prarit.bos.redhat.com>
Patchwork-id: 24964
O-Subject: [RHEL5 PATCH 4/27] compat.h cleanup: e1000 and e1000e driver changes
Bugzilla: 546740
RH-Acked-by: Andy Gospodarek <gospo@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

>From d928e9485803a8e9edee31fc1a9bd27614f029e2 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 10 May 2010 09:22:13 -0400
Subject: [PATCH] e1000 and e1000e driver changes

Resolves 546740.

diff --git a/drivers/net/e1000/e1000.h b/drivers/net/e1000/e1000.h
index 2af031f..f5c7b91 100644
--- a/drivers/net/e1000/e1000.h
+++ b/drivers/net/e1000/e1000.h
@@ -79,7 +79,6 @@
 
 struct e1000_adapter;
 
-#include "e1000_compat.h"
 #include "e1000_hw.h"
 
 #ifdef DBG
diff --git a/drivers/net/e1000/e1000_compat.h b/drivers/net/e1000/e1000_compat.h
deleted file mode 100644
index 08beddb..0000000
--- a/drivers/net/e1000/e1000_compat.h
+++ /dev/null
@@ -1,6 +0,0 @@
-#ifndef _E1000_COMPAT_H_
-#define _E1000_COMPAT_H_
-
-#define ETH_FCS_LEN	4		/* Octets in the FCS */
-
-#endif /* _E1000_COMPAT_H_ */
diff --git a/drivers/net/e1000e/e1000.h b/drivers/net/e1000e/e1000.h
index 1bb922a..e17ec64 100644
--- a/drivers/net/e1000e/e1000.h
+++ b/drivers/net/e1000e/e1000.h
@@ -37,7 +37,6 @@
 #include <linux/io.h>
 #include <linux/netdevice.h>
 
-#include "e1000_compat.h"
 #include "hw.h"
 
 struct e1000_info;
diff --git a/drivers/net/e1000e/e1000_compat.h b/drivers/net/e1000e/e1000_compat.h
deleted file mode 100644
index 14e42fe..0000000
--- a/drivers/net/e1000e/e1000_compat.h
+++ /dev/null
@@ -1,21 +0,0 @@
-#ifndef __E1000E_COMPAT_H__
-#define __E1000E_COMPAT_H__
-
-#include <linux/if_vlan.h>
-#include <linux/pci.h>
-
-#define ETH_FCS_LEN               4
-
-static inline struct net_device *vlan_group_get_device(struct vlan_group *vg,
-						       int vlan_id)
-{
-	return vg->vlan_devices[vlan_id];
-}
-
-static inline void vlan_group_set_device(struct vlan_group *vg, int vlan_id,
-					 struct net_device *dev)
-{
-	vg->vlan_devices[vlan_id] = NULL;
-}
-
-#endif 
